src/manifolds/Multinomial.jl

```diff
@@ -0,0 +1,97 @@
+@doc raw"""
+    MultinomialMatrices{n,m} <: AbstractPowerManifold{ℝ}
+
+The multinomial manifold consists of `m` column vectors, where each column is of length
+`n` and unit norm, i.e.
+
+````math
+\mathcal{MN}(n,m) \coloneqq \bigl\{ p ∈ ℝ^{n×m}\ \big|\ p_{i,j} > 0 \text{ for all } i=1,…,n, j=1,…,m \text{ and } p^{\mathrm{T}}\mathbb{1}_m = \mathbb{1}_n\bigr\},
+````
+where $\mathbb{1}_k$ is the vector of length $k$ containing ones.
+
+This yields exactly the same metric as
+considering the product metric of the probablity vectors, i.e. [`PowerManifold`](@ref) of the
+$(n-1)$-dimensional [`ProbabilitySimplex`](@ref).
+
+The [`ProbabilitySimplex`](@ref) is stored internally within `M.manifold`, such that all functions of
+[`AbstractPowerManifold`](@ref) can be used directly.
+
+# Constructor
+
+    MultinomialMatrices(n,m)
+
+Generate the manifold of matrices $\mathbb R^{n×m}$ such that the $m$ columns are
+discrete probability distributions, i.e. sum up to one.
+"""
+struct MultinomialMatrices{N,M,S} <:
+       AbstractPowerManifold{ℝ,ProbabilitySimplex{S},ArrayPowerRepresentation} where {N,M}
+    manifold::ProbabilitySimplex{S}
+end
+
+MultinomialMatrices(n::Int, m::Int) =
+    MultinomialMatrices{n,m,n - 1}(ProbabilitySimplex(n - 1))
+
+^(M::ProbabilitySimplex{N}, m::Int) where {N} =
+    MultinomialMatrices{manifold_dimension(M) + 1,m,N}(M)
+
+@doc raw"""
+    check_manifold_point(M::MultinomialMatrices,p)
```

Comment:
```suggestion
    check_manifold_point(M::MultinomialMatrices, p)
```

Reply:
Interesting, I even ran formatter, but thanks for noticing.

Reply:
I think the formatter doesn't format docstrings.

Reply:
Ah, thanks for noting that, I even got a little lazy, but will carefully still check docs then.